apps/backend/db_patches/0148_CreateUpdateUserIdFunc.sql

```diff
@@ -0,0 +1,77 @@
+          UPDATE FAP_ASSIGNMENTS SET FAP_MEMBER_USER_ID = P_NEW_USER_ID WHERE FAP_MEMBER_USER_ID = P_OLD_USER_ID; 
+          UPDATE FAP_MEETING_DECISIONS SET SUBMITTED_BY = P_NEW_USER_ID WHERE SUBMITTED_BY = P_OLD_USER_ID; 
+          UPDATE FAPS SET FAP_CHAIR_USER_ID = P_NEW_USER_ID WHERE FAP_CHAIR_USER_ID = P_OLD_USER_ID; 
+          UPDATE FAPS SET FAP_SECRETARY_USER_ID = P_NEW_USER_ID WHERE FAP_SECRETARY_USER_ID = P_OLD_USER_ID; 
+          UPDATE FEEDBACKS SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE GENERIC_TEMPLATES SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE INSTRUMENTS SET MANAGER_USER_ID = P_NEW_USER_ID WHERE MANAGER_USER_ID = P_OLD_USER_ID; 
+          UPDATE INTERNAL_REVIEWS SET ASSIGNED_BY = P_NEW_USER_ID WHERE ASSIGNED_BY = P_OLD_USER_ID; 
+          UPDATE INTERNAL_REVIEWS SET REVIEWER_ID = P_NEW_USER_ID WHERE REVIEWER_ID = P_OLD_USER_ID; 
+          UPDATE PDF_TEMPLATES SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE PREDEFINED_MESSAGES SET LAST_MODIFIED_BY = P_NEW_USER_ID WHERE LAST_MODIFIED_BY = P_OLD_USER_ID; 
+          UPDATE PROPOSALS SET PROPOSER_ID = P_NEW_USER_ID WHERE PROPOSER_ID = P_OLD_USER_ID; 
+          UPDATE QUESTIONARIES SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE REDEEM_CODES SET CLAIMED_BY = P_NEW_USER_ID WHERE CLAIMED_BY = P_OLD_USER_ID; 
+          UPDATE REDEEM_CODES SET CREATED_BY = P_NEW_USER_ID WHERE CREATED_BY = P_OLD_USER_ID; 
+          UPDATE REDEEM_CODES SET PLACEHOLDER_USER_ID = P_NEW_USER_ID WHERE PLACEHOLDER_USER_ID = P_OLD_USER_ID; 
+          UPDATE SAMPLES SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE SCHEDULED_EVENTS SET LOCAL_CONTACT = P_NEW_USER_ID WHERE LOCAL_CONTACT = P_OLD_USER_ID; 
+          UPDATE SHIPMENTS SET CREATOR_ID = P_NEW_USER_ID WHERE CREATOR_ID = P_OLD_USER_ID; 
+          UPDATE TECHNICAL_REVIEW SET REVIEWER_ID = P_NEW_USER_ID WHERE REVIEWER_ID = P_OLD_USER_ID; 
+          UPDATE VISITS SET VISIT_ID = P_NEW_USER_ID WHERE VISIT_ID = P_OLD_USER_ID; 
+          UPDATE VISITS SET TEAM_LEAD_USER_ID = P_NEW_USER_ID WHERE TEAM_LEAD_USER_ID = P_OLD_USER_ID; 
+          --Update TABLES row with OLD USER ID with NEW USER ID 
+          --where the row is not overlap with NEW USER ID
+          UPDATE FAP_REVIEWERS SET USER_ID = P_NEW_USER_ID WHERE USER_ID = P_OLD_USER_ID 
+            AND FAP_ID NOT IN ( SELECT FAP_ID FROM  FAP_REVIEWERS  WHERE USER_ID = P_NEW_USER_ID ) ;
+          UPDATE FAP_REVIEWS SET USER_ID = P_NEW_USER_ID WHERE USER_ID = P_OLD_USER_ID 
+            AND PROPOSAL_PK NOT IN ( SELECT PROPOSAL_PK FROM  FAP_REVIEWS  WHERE USER_ID = P_NEW_USER_ID ) ;
+          --UPDATE FAP_SECRETARIES SET USER_ID = P_NEW_USER_ID WHERE USER_ID = P_OLD_USER_ID 
+          --AND FAP_ID NOT IN ( SELECT  FAP_ID FROM  FAP_SECRETARIES  WHERE USER_ID = P_NEW_USER_ID ) ;
```

Comment:
Why is this UPDATE statement commented out?

Reply:
USERS table entry is removed and the lines are uncommented.